### ISSUE TITLE
Cannot install anything

### ISSUE THREAD

**Issue by lud-wj** [flox/flox#22]
I've just installed flox but for some reason it wants to access my github CLI data:

```
flox install cowsay --debug     
[DEBUG] [flox::config] `$FLOX_CONFIG_HOME` not set, using "/home/lud/.config/flox/"
[DEBUG] [flox::config] `FLOX_CONFIG_HOME` not set, using "/home/lud/.config/flox/"
[WARN] [flox::utils::init::channels] Did not find "/home/lud/.config/flox/floxUserMeta.json", continuing without user channels
[ERROR] [flox] Could not read `gh` config file
    
    Caused by:
        github.com: missing field `oauth_token` at line 2 column 5

```

What can I do ? I do not have neither do I need a github access token.

**Comment by mootpt** [comment]
this has happened to me when I had corrupt gh cli config. I ended up blowing away the config directory `rm -rf ~/.config/gh`

**Comment by aakropotkin** [comment]
Thanks for the suggestion @mootpt.

@lud-wj let me know if recreating the `~/.config/gh` file works.

Did you follow the `gh auth` setup guide in our docs? If not go ahead and make sure you've set that up: https://floxdev.com/docs/share-environments/#flox-auth

Let me know if either of those help and if not I can help you debug this.

**Comment by lud-wj** [comment]
Hello all, 

Thank you for your answers.

This is the content of my `~/.config/gh/hosts.yml`:

```
github.com:
    user: lud-wj
    git_protocol: ssh
```

I cannot let a third party tool access my github account because it is my company computer and that github account has access to a lot of private/internal repositories.

I do not need to sync my flox environments in my github account automatically, they will be probably shared with coworkers at some point (and in that case we may use that flox feature if approved).

**Comment by aakropotkin** [comment]
This is a great question and one that has been a part of some project roadmap discussions this week.

My understanding is that currently the git repo is necessary. This may change in the future though.

**Comment by mkenigs** [comment]
> This is the content of my `~/.config/gh/hosts.yml`:
> 
> ```
> github.com:
>     user: lud-wj
>     git_protocol: ssh
> ```
> 

Thanks!

> 
> I added an empty `oauth_token:` pair in the file, and now flox seems to run, but it immediately creates a floxmeta repository. Is there a way to disable using github or is it mandatory for flox to run?

Do you know at what point it created the floxmeta repo? If it's creating that before a `flox push` that may be a bug

**Comment by mkenigs** [comment]
> I am looking for a replacement to asdf where we could just keep a flox env file in a repo and let anyone start a shell with the right tools for this repo (Elixir/Erlang).

That's a use case we definitely want to support so we'd love any feedback you have on bugs or usability!

**Comment by lud-wj** [comment]
> Do you know at what point it created the floxmeta repo? If it's creating that before a `flox push` that may be a bug

@mkenigs from what I remember creating the repo seemed to be the first thing flox did when I ran `flox install cowsay`.

Now that I have deleted the created repo and reverted gh config to it's original state, it does not seem to want to create the repositiory anymore:

```
~ ❯❯❯ flox install cowsay
Installed 'cowsay' package(s) into 'default' environment.
~ ❯❯❯ cowsay                  
zsh: command not found: cowsay

```

**Comment by lud-wj** [comment]
Ah yes, environment it is!

> If we're creating the repo without prompting that's 100% a bug so let us know if you can reproduce that.

Well it will systematically create the repo when I call the command. 
What should I do to completely purge flox from my system besides simple "desinstall", do I can try to reinstall from scratch and see if always create the repository?

**Comment by aakropotkin** [comment]
Which install method did you use?

I believe that the `apt` and OSX installs can essentially just use the associated uninstaller. If you had used the `nix` installs `nix profile remove` should do the trick. Our docs cover this in more detail under the "Uninstall `flox`" section.

Aside from that you may want to wipe out `~/.cache/flox` and `~/.config/flox`.

**Comment by lud-wj** [comment]
`apt` IIRC. 

Thank you I'll try to do a full cleanup and let you know.

**Comment by aakropotkin** [comment]
Did you have any luck with the full cleanup?

**Comment by lud-wj** [comment]
No sorry I kept rescheduling it. I just dit it right now without having nix installed before and it looks like the simple  [Debian guide](https://floxdev.com/docs/#install-flox) works properly, cause I can install cowsay wihtout having flox trying to mess with my github.

I did have these messages in the console though, not sure if it is related:

```
avertissement : la HEAD distante réfère à une référence non existante, impossible de l'extraire
warning: input 'flox' has an override for a non-existent input 'nixpkgs-flox'
```

The first message means _the distant HEAD points to a non existing reference, cannot extract_. It is a Git message.
